index.html

```diff
@@ -1,277 +1,275 @@
-<style>
-  body {
-    background-color: rgb(91, 146, 229);
-  }
-
-  .logo {
-    width: 100%;
-  }
-  
-  article {
-    max-width: 500px;
-    margin: 0 auto;
-    color: rgb(255, 255, 255);
-  }
-  
-  h2 {
-    font-family: sans-serif;
-    padding-bottom: 10px;
-		border-bottom: 3px solid rgb(255, 255, 255);
-  }
-  
-  p {
-    line-height: 20px;
-  }
-</style>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="css/styles.css">
+    <!-- <link rel="icon" type="image/png" href="assets/...png"> -->
+    <meta name="viewport" content="width=device-width">
+
+    <title>Digitizing Human Rights: Education</title>
+    <meta name="description" content="Digitizing Human Rights: Educations">
+    <meta name="author" content="Cloris Dong, Silvia Craig, Preranaa Srinivas">
+
+    <script src="js/p5.js"></script>
+    <script src="js/p5.sound.js"></script>
+    <script src="js/Education.js"></script>
+
+    <style>
+      html, body{
+       margin: 0;
+       padding: 0;
+       height: 100%;
+       width: 100%;
+      }
+
+      canvas {
+       display: block;
+       height: 100%;
+       width: 100%;
+      }
+
+      html {
+        background: rgb(61,173,219);
+        background: linear-gradient(360deg, rgba(61,173,219,1) 16%, rgba(77,131,212,1) 100%) fixed;
+        background-repeat: no-repeat;
+      }
+
+      .udhrlogo {
+        display: block;
+        margin-left: auto;
+        margin-right: auto;
+        margin-bottom: 0px;
+        padding-top: 15px;
+        width: 20%;
+      }
+
+      <!-- article {
+        max-width: 500px;
+        margin: 0 auto;
+        color: rgb(255, 255, 255);
+      } -->
+
```

Comment:
the `<!-- syntax ->` is used to comment out code in HTML, but this section of ur code is CSS so u need to use the `/* syntax */`